account_cash_invoice/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_cash_pay_invoice,access_cash_pay_invoice,model_cash_pay_invoice,base.group_user,1,1,1,1
```

Comment:
Too broad permission. Please restrict to the real ones, which are those that do reconciliation.